index.js

```diff
@@ -119,12 +128,15 @@ module.exports = {
         outdent;
     }
 
+
     return {
       name,
       modulePrefix: name,
       namespace,
-      addonName,
-      addonNamespace,
+      addonInfo,
+      testAppInfo,
+      addonName: addonInfo.name.dashed,
+      addonNamespace: addonInfo.name.classified,
```

Comment:
I think we must expose the location as `addonDirectory` (so it aligns with https://github.com/ember-cli/ember-cli/blob/master/blueprints/addon/index.js#L152), and use it here: https://github.com/embroider-build/addon-blueprint/blob/main/files/CONTRIBUTING.md?plain=1#L6